components/external_storage/examples/scli.rs

```diff
@@ -41,9 +41,12 @@ pub struct Opt {
     /// Credential file path. For S3, use ~/.aws/credentials.
     #[structopt(short, long)]
     credential_file: Option<String>,
+    /// Remote endpoint
+    #[structopt(short, long)]
+    endpoint: Option<String>,
     /// Remote region.
     #[structopt(short, long)]
-    region: Option<u64>,
+    region: Option<String>,
```

Comment:
Isn't it a number?

Reply:
it's a string, something like `us-east-2`: https://docs.aws.amazon.com/general/latest/gr/rande.html

Reply:
It's not the region in TiKV.